data/clients.json

```diff
@@ -737,7 +742,7 @@
     },
     {
         "doap": null,
-        "last_renewed": "2019-07-16T20:55:15",
+        "last_renewed": "2021-02-09T00:00:00",
```

Comment:
Please remove it entirely: https://github.com/zom/Zom-iOS-XMPP
It not even using XMPP anymore.